src/form/index.tsx

```diff
@@ -69,6 +71,13 @@ export default factory(function Form({
 		classes: themedCss.root
 	};
 
+	if (props.name) {
```

Comment:
i think this can just be used as-is on the form render, if it's undefined it'll be ignored by the renderer

Reply:
You are correct, just tested this out. This if check unnecessary.

Reply:
👍